### PR TITLE
[12.0] [MIG] account_analytic_sequence

### PR DESCRIPTION
**Pull request by bjeficent**
Superseed #199

### CODE REVIEW

File: account_analytic_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Analytic Sequence',
+    'summary': """
+        Restore the analytic account sequence""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu',
```
Comment (alexey-pelykh) [inline account_analytic_sequence/__manifest__.py]: This should point to OCA repository as per https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11

File: account_analytic_sequence/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import test_analytic_sequence
```
Comment (alexey-pelykh) [inline account_analytic_sequence/tests/__init__.py:3]: A license notice and a newline would be great!

File: account_analytic_sequence/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from . import account_analytic_account
```
Comment (alexey-pelykh) [inline account_analytic_sequence/models/__init__.py:3]: License notice would be awesome

File: account_analytic_sequence/README.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Account Analytic Sequence
```
Comment (alexey-pelykh) [inline account_analytic_sequence/README.rst:2]: Splitting readme as per https://github.com/OCA/maintainer-tools/tree/master/template/module/readme would be great!

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Review by alexey-pelykh — APPROVED** [review]

**Comment by ntodorova** [comment]
in progress

File: account_analytic_sequence/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from . import models
```
Comment (ntodorova) [inline account_analytic_sequence/__init__.py:3]: I though that the checks should fail if there is no new line? May be you can add one

File: account_analytic_sequence/data/sequence.xml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
```
Comment (ntodorova) [inline account_analytic_sequence/data/sequence.xml:3]: do we have to put license file in the views?

File: account_analytic_sequence/data/sequence.xml [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+<odoo noupdate="1">
+
+    <record model="ir.sequence" id="analytic_account_code_seq">
```
Comment (ntodorova) [inline account_analytic_sequence/data/sequence.xml]: may be put id before model?

File: account_analytic_sequence/tests/__init__.py [hunk 8]
```diff
@@ -0,0 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from . import test_analytic_sequence
```
Comment (ntodorova) [inline account_analytic_sequence/tests/__init__.py:3]: a new line?

**Review by ntodorova — COMMENTED** [review]
I did a code review and it looks fine, I just added some small suggestions for improvement.

File: account_analytic_sequence/tests/__init__.py (hunk 8 above)
Comment (alexey-pelykh) [reply account_analytic_sequence/tests/__init__.py:3]: One line between license and import is enough, see https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: account_analytic_sequence/data/sequence.xml (hunk 6 above)
Comment (alexey-pelykh) [reply account_analytic_sequence/data/sequence.xml:3]: There's no requirement on that, yet this file is still some sort of code, and should contain license/copyright notice

File: account_analytic_sequence/__init__.py (hunk 5 above)
Comment (alexey-pelykh) [reply account_analytic_sequence/__init__.py:3]: One line between license and import is enough, see https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Comment (ntodorova) [reply account_analytic_sequence/__init__.py:3]: ok

File: account_analytic_sequence/tests/__init__.py (hunk 8 above)
Comment (ntodorova) [reply account_analytic_sequence/tests/__init__.py:3]: ok

File: account_analytic_sequence/data/sequence.xml (hunk 7 above)
Comment (bjeficent) [reply account_analytic_sequence/data/sequence.xml]: Update done.

**Comment by AaronHForgeFlow** [comment]
@ntodorova Do you think your suggestions have been attended?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Pull request merged**